tests/modules/snpdists/test.yml

```diff
@@ -0,0 +1,9 @@
+## TODO nf-core: Please run the following command to build this file:
+#                nf-core modules create-test-yml snpdists
```

Comment:
leftover todo statement

Reply:
Whoops!

Fixed with https://github.com/nf-core/modules/pull/697/commits/ac28108107fba07809c8c0257cbc6b0bc3826e6b 

Thanks for the review @grst !